src/components/pages/CaseTable/CaseTable.js

```diff
@@ -398,71 +411,28 @@ export default function CaseTable(props) {
           key="1"
           className="exportBtn"
           icon={<FeatherIcon icon="download" />}
```

Comment:
We have a download icon instead of using Feather icon, I want to see if this matches the site first